### ISSUE TITLE
train code

### ISSUE THREAD

**Issue by tom666tom666** [GGGHSL/InfoSwap-master#1]
Do you have a plan to release the training code?

**Comment by GGGHSL** [comment]
Sorry, we don't have this plan at the moment. I suggest you write your own training code, as the discriminator is provided and the loss functions are detailed in the paper.
